sinfonier-backend-api/apiRequests/RequestHandler.py

```diff
@@ -3,13 +3,17 @@
 
 class Error(dict):
     def __init__(self, msg):
-        super(Error).__init__()
+        #super(Error).__init__()
```

Comment:
You have got inheritance from dict so, it's a good idea to initialized before to use isn't?. I added this object because sometimes we returned a dict and others a string, this was for unificate the error to the same type.